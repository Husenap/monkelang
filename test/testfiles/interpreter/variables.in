monke has number bananas is 42
monke say "monke has " + bananas + " bananas."

monke has number bananas
monke say "monke has " + bananas + " bananas."

bananas is 5
monke say "monke has " + bananas + " bananas."

monke has number a = 3
monke has number b = 6
monke say a + b

monke has number a is 1
monke say "a is " + a
a is 2
monke say "a is " + a
monke say a is 3


monke has number a
monke has number b
monke has number c
a is b is c is 42
monke say a + b + c

