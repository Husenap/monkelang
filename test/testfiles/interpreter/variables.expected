monke has 42 bananas.
monke has nil bananas.
monke has 5 bananas.
9
a is 1
a is 2
3
126
